src/components/TextField/TextField.tsx

```diff
@@ -21,6 +21,7 @@ import { NOOP } from "../../utils/function-utils";
 import { ComponentDefaultTestId } from "../../tests/constants";
```

Comment:
Please add a snapshot test for testing
1. TextField with description 
2.  Description with different icon